Ansible/play_file

```diff
@@ -0,0 +1,91 @@
+---
+- name: Install To Do App on new webserver
+  hosts: managed_hosts
+  remote_user: ec2-user
+
+  vars_prompt:
+    - name: api_key
+      prompt: What is the api key?
+      private: no
+
+    - name: api_token
+      prompt: What is the api token?
+      private: no
```

Comment:
This is fine here because we're only handling it directly on the server, but worth noting that this value _is_ secret, so it might be worth marking it as such. For example, if we were to later bring this playbook so that it's run as part of a pipeline, it would be easy to miss that this might mean the api_token ends up in pipeline logs and visible to users who shouldn't have access